lib/tensorflow/graph.rb

```diff
@@ -96,11 +96,11 @@ def make_attr_value(attribute_type, value)
   end
 
   TYPE2ENUM = {
-    DT_FLOAT: Tensorflow::TF_FLOAT,
-    DT_DOUBLE: Tensorflow::TF_DOUBLE,
-    DT_INT64: Tensorflow::TF_INT64,
-    DT_STRING: Tensorflow::TF_STRING,
-    DT_COMPLEX128: Tensorflow::TF_COMPLEX128
+    DT_FLOAT: Tensorflow::Internal::TF_FLOAT,
+    DT_DOUBLE: Tensorflow::Internal::TF_DOUBLE,
+    DT_INT64: Tensorflow::Internal::TF_INT64,
+    DT_STRING: Tensorflow::Internal::TF_STRING,
+    DT_COMPLEX128: Tensorflow::Internal::TF_COMPLEX128
```

Comment:
you don't need `include Tensorflow`.

```ruby
module MyModule
  class A
  end

  class B
    def test
      p A.new # no need to write MyModule::A, because their are visible to each other
    end
  end
end

MyModule::B.new.test # works
```